testnet-contracts/contracts/Oracle.sol

```diff
@@ -0,0 +1,243 @@
+pragma solidity ^0.5.0;
+
+import "../../node_modules/openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "./Valset.sol";
+import "./CosmosBridge.sol";
+
+contract Oracle {
+
+    using SafeMath for uint256;
+
+    /*
+    * @dev: Public variable declarations
+    */
+    CosmosBridge public cosmosBridge;
+    Valset public valset;
+    address public operator;
+
+    // Tracks the number of OracleClaims made on an individual BridgeClaim
+    mapping(uint256 => address[]) public oracleClaimValidators;
+    mapping(uint256 => mapping(address => bool)) public hasMadeClaim;
+
+    /*
+    * @dev: Event declarations
+    */
+    event LogNewOracleClaim(
+        uint256 _bridgeClaimID,
```

Comment:
Again, we're kind of mixing the concept of _bridgeClaimID and _prophecyID. They are the same thing conceptually and I think we can just call it bridgeProphecyID everywhere?

Reply:
https://github.com/cosmos/peggy/pull/82/commits/60cb52d4b6baf69bc7e237ba9839b1a9e2fae1a2